packages/core/selectors/getProviders.test.js

```diff
@@ -0,0 +1,5 @@
+import getProviders from './getProviders';
+
+it('should get empty array', () => {
```

Comment:
Still trying to understand how to test this. Tried a few variations with no luck.